source/_integrations/halohome.markdown

```diff
@@ -0,0 +1,25 @@
+---
+title: HALO Home
+description: Instructions on integrating HALO Home smart lights into Home Assistant.
+ha_category:
+  - Light
+ha_release: 2022.1
+ha_iot_class: Assumed State
+ha_codeowners:
+  - '@nayaverdier'
+ha_domain: halohome
+ha_config_flow: true
+ha_platforms:
+  - light
+---
+
+Integrates [HALO Home](https://www.cooperlighting.com/global/brands/halo-home) smart lights into Home Assistant.
+
+Supports brightness and color temperature for Bluetooth HALO Home lights.
+
+Devices are read from the HALO Home API upon setup and stored by this
+integration to allow devices to be controlled even when the internet is not
+accessible. Because of this, if devices are added or removed, the integration
+must be reconfigured.
```

Comment:
Without looking at the code at this point, that seems weird. A user should not have to reconfigure to add devices. Instead, it should just pick them up when the connection has been restored.

Reply:
Would it be acceptable to check for new devices when the integration is (re)loaded, or is there another preferred solution?

Reply:
> Would it be acceptable to check for new devices when the integration is (re)loaded, or is there another preferred solution?

Yeah, that is possible, or maybe poll every now and then to check the state. Having to provide instructions for these things, generally indicate something is a bit off / could be improved for the end-user.